src/Host/Client/Impl/rtvs/R/eval.R

```diff
@@ -67,6 +67,17 @@ describe_object <- function(obj, res, fields, repr = NULL) {
       res$dim <- as.list(dim);
     }
   }
+  
+  if (field('to_df')) {
+    can_coerce_to_df <- function(obj){
+      df_test <- function(objclass){
+        res <- getS3method('as.data.frame', objclass, optional = TRUE);
+        if (is.null(res)) { FALSE } else { TRUE }
+      }
+      any(as.logical(lapply(class(obj), df_test)), na.rm = TRUE);
```

Comment:
Ah, I see, it always returns a list. `sapply` or `vapply` instead of `lapply` should do the trick.